doc/releases/changelog-0.28.0.md

```diff
@@ -292,234 +287,62 @@
+  [(#3391)](https://github.com/PennyLaneAI/pennylane/pull/3391)
 
   ```pycon
-  >>> circuit(a)
-  ((array(0.904), array([0.952, 0.   , 0.   , 0.048])),
-   (array(0.915), array([0.9575, 0.    , 0.    , 0.0425])),
-   (array(0.902), array([0.951, 0.   , 0.   , 0.049])))
-  >>> cost(a)
-  array([[0.91      , 0.955     , 0.        , 0.        , 0.045     ],
-         [0.895     , 0.9475    , 0.        , 0.        , 0.0525    ],
-         [0.90666667, 0.95333333, 0.        , 0.        , 0.04666667]])
-  >>> qml.jacobian(cost)(a)
-  array([[-0.37      , -0.185     ,  0.        ,  0.        ,  0.185     ],
-         [-0.409     , -0.2045    ,  0.        ,  0.        ,  0.2045    ],
-         [-0.37133333, -0.18566667,  0.        ,  0.        ,  0.18566667]])
+  >>> pw = qml.pauli.PauliWord({0: 'X', 1: 'Y'})
+  >>> print(pw.operation())
+  PauliX(wires=[0]) @ PauliY(wires=[1])
+  >>> print(pw.hamiltonian())
+    (1) [X0 Y1]
+  >>> ps = qml.pauli.PauliSentence({pw: -1.23})
+  >>> print(ps.operation())
+  -1.23*(PauliX(wires=[0]) @ PauliY(wires=[1]))
+  >>> print(ps.hamiltonian())
+    (-1.23) [X0 Y1]
   ```
+* A `sum_expand` function has been added for tapes, which splits a tape measuring a `Sum` expectation into mutliple tapes of summand expectations, and provides a function to recombine the results.
+  [(#3230)](https://github.com/PennyLaneAI/pennylane/pull/3230)
+  
+<h4>(Experimental) More interface support for multi-measurement and gradient output types 🧪</h4>
 
-* The TensorFlow interface for the new return types now supports devices with shot vectors.
+* The autograd and Tensorflow interfaces now support devices with shot vectors when `qml.enable_return()` has been called.
+  [(#3374)](https://github.com/PennyLaneAI/pennylane/pull/3374)
   [(#3400)](https://github.com/PennyLaneAI/pennylane/pull/3400)
 
-  Example with a single measurement:
+  Here is an example using Tensorflow:
 
   ```python
-  dev = qml.device("default.qubit", wires=1, shots=[1000, 2000, 3000])
-
-  @qml.qnode(dev, diff_method="parameter-shift", interface="tf")
-  def circuit(a):
-      qml.RY(a, wires=0)
-      qml.RX(0.2, wires=0)
-      return qml.expval(qml.PauliZ(0))
-  ```
+  import tensorflow as tf
 
-  ```
-  >>> qml.enable_return()
-  >>> a = tf.Variable(0.4)
-  >>> with tf.GradientTape() as tape:
-  ...     res = circuit(a)
-  ...     res = tf.stack(res)
-  ...
-  >>> res
-  <tf.Tensor: shape=(3,), dtype=float64, numpy=array([0.902     , 0.904     , 0.89533333])>
-  >>> tape.jacobian(res, a)
-  <tf.Tensor: shape=(3,), dtype=float64, numpy=array([-0.365     , -0.3765    , -0.37533333])>
-  ```
-
-  Example with multiple measurements:
+  qml.enable_return()
 
-  ```python
   dev = qml.device("default.qubit", wires=2, shots=[1000, 2000, 3000])
 
   @qml.qnode(dev, diff_method="parameter-shift", interface="tf")
```

Comment:
On line 356, please can we add:
`>>> a = tf.Variable(0.4)`

Reply:
Good catch!